jsonnet/dashboards/k8s/memory-namespace-overview.libsonnet

```diff
@@ -44,7 +44,7 @@ local graphPanel = grafana.graphPanel;
           sort={ col: 8, desc: true },
           styles=[
             { pattern: 'Time', type: 'hidden' },
-            { alias: 'Pods', pattern: 'Value #A', type: 'number' },
+            { alias: 'Pods', pattern: 'Value', type: 'number' },
```

Comment:
This doesn't work. there are also \#B \#C...

Reply:
Fixed.